src/internal/kopia/base_finder.go

```diff
@@ -206,7 +206,7 @@ func (b *baseFinder) findBasesInSet(
 	ctx context.Context,
 	reason identity.Reasoner,
 	metas []*manifest.EntryMetadata,
-) (*backupBase, *backupBase, error) {
+) (*BackupBase, *BackupBase, error) {
```

Comment:
Nit: I know it's not part of the changes here, per se, but return values like this make me just a little uneasy since it's difficult to distinguish which value is which without referencing either the func implementation or other code.  Could we pack these into a struct as a way of labeling them?

Reply:
I'll pick this up in a later PR. I'd like to get this one done and there's already quite a bit going on here